api/src/client/ApiFormationClient.test.ts

```diff
@@ -567,13 +568,22 @@ describe("ApiFormationClient", () => {
           formationFormData
         );
 
-        await client.form(userData, "hostname.com/form-business");
+        const foreignGoodStandingFile = generateInputFile({
+          fileType: "PNG",
+          base64Contents: "this is my string",
+        });
```

Comment:
I don't think we need to specify the fileType and base64Contents? We're never testing for them later on. I tend to push for reducing/removing any extraneous info from tests so that everything that is present has a reason to be. I think here we could just let the random generator do its work, we don't care the exact values.

Reply:
Fair point. Included!